### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by Kaweesi-Matia**
- [ ] Splash screen

- A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.

- [ ]  Sign up and log in pages

- The user should be able to register in the app with full name, email and password (all mandatory).

- The user can log into the app using email and password.

- If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).

 

- [ ] Home page (categories page)

- When the user logs in, they are presented with the categories page.

- For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.

- When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.

- There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

 

- [ ] Transactions page

- For a given category, the list of transactions is presented, ordered by the most recent.

- At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).

- There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.

- When the user clicks on the "Back" button (<), the user navigates to the home page.

 

- [ ] "Add a new category" page

- The user fills out a form to create a new category, indicating their name and icon (both mandatory).
## Below is the description of the PR
- The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.

- When the user clicks on the "Back" button (<), the user navigates to the home page.

 

- [ ] "Add a new transaction" page

- [ ]  Create unit and integration tests for all the most important components of your RoR application.

 

- [ ] Modify Readme file

 

- [ ] Add linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,192 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> [Budget App Demo](budget-app-pearl-ten.vercel.app)
+
```
Comment (thecodechaser) [inline README.md]: - [ ] The live demo link in the README file is broken, Please make sure the project is deployed and accessible online as requested in the project requirements and that the live demo link in the README file works as expected.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Kaweesi-Matia👋, 

### You are doing a marvelous job on the RoR capstone project and this is one of the best projects I have seen so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Highly organized code. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- All tests are passed. ✔️
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Kaweesi-Matia) [reply README.md]: Add a live demo link in the README file as required by the reviewer

**Review by Wes-Isaac — APPROVED** [review]
Hi @Kaweesi-Matia, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**